index.html

```diff
@@ -0,0 +1,571 @@
+
+## Prefixing
+
+With the exception of a single root directive, core feature specifications **must** prefix all schema elements they introduce. The prefix:
+1. **must** match the default name of the feature as derived from the feature's specification URL,
+1. **must** be a string of characters valid within GraphQL names, and
+2. **must not** contain the core namespace separator, which is two underscores (`"__"`).
+
+Prefixed names consist of the name of the feature, followed by two underscores, followed by the name of the element (which can be any valid GraphQL identifier). For instance, the `core` specification (which you are currently reading) introduces elements named [`core__FeatureUrl`](#core__featureurl) and `core__export`(#core__export).
+
+A feature's *root directive* is an exception to the prefixing requirements. Feature specifications **may** introduce a single directive which carries only the name of the feature, with no prefix required. For example, the `core` specification introduces a `@core` directive. This directive has the same name as the feature ("`core`"), and so requires no prefix.
+
+```graphql
+#::[Using the `@core` directive with a spec's default name]
+schema @core(feature: "https://spec.example.com/example/v1.0") {
+  query: Query
+}
+
+type User {
+  name: String @example(data: ITEM)
+}
+
+# An enum used to provide structured data to the example spec.
+# It is prefixed with the name of the spec.
+enum example__Data {
+  ITEM
+}
+
+directive @example(data: example__Data) on FIELD
+```
+
+The prefix **must not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which must be prefixed
+
+Feature specs **must** prefix the following schema elements:
+- the names of any object types, interfaces, unions, enums, or input object types defined by the feature
+- the names of any fields the spec introduces on *foreign types* defined in a different feature
+- the names of any arguments the spec introduces on *foreign directives and fields* defined by a different feature
+- the names of any directives introduced in the spec, with the exception of the *root directive*, which must have the same name as the feature
+
+<view-of src=prefixing.graphql
+  title='Prefixing examples'
+></view-of>
+
+## Versioning
+
+Specs are versioned with a **subset** of a [Semantic Version Number](https://semver.org/spec/v2.0.0.html) containing only the `major` and `minor` parts. Thus, specifications **should** provide a version of the form `‹major›.‹minor›`, where both integers >= 0.
```

Comment:
Maybe word this as "Specs are versioned with a two-part version number, inspired by Semantic Versioning but without the patch, pre-release, and build metadata parts."  (Specifically defining it as a subset of the other spec feels weird... it's more of a substring... and the spec is called Semantic Versioning.)